ui-tests/src/test/java/io/syndesis/qe/pages/integrations/importt/ImportIntegration.java

```diff
@@ -21,7 +22,8 @@
     private static final class Element {
         public static final By ROOT = By.className("pf-c-page__main");
         public static final By FILE_INPUT = By.cssSelector("input[type='file']");
-        public static final By FINISHED_PROGRESS_BAR = By.className("pficon-ok");
+        public static final By FINISHED_PROGRESS_BAR =
+            ByUtils.dataTestIdContainsSubstring("dnd-file-chooser-successfully-imported-", 1);
```

Comment:
`ByUtils#containsDataTestId` can be used as well and doesn't require magic constant